examples/resources/awscc_sso_assignment/use_new_permission_set.tf

```diff
@@ -0,0 +1,59 @@
+// Note: Currently there is no data source for fetching the IAM Identity Center (formerly AWS SSO)
+//instance arn in the awscc provider so we must use both the aws and awscc providers.
+terraform {
+  required_providers {
+    aws = {
+      source = "hashicorp/aws"
+    }
+    awscc = {
+      source = "hashicorp/awscc"
+    }
+  }
+}
+
+data "aws_ssoadmin_instances" "example" {} // fetch IAM Identity Center instance arn
+
+// create new permission set
+resource "awscc_sso_permission_set" "example" {
+  instance_arn = data.aws_ssoadmin_instances.example.arns[0] // reference existing IAM IDC instance by arn
+  name         = "ExamplePermissionSet"                      // add desired name for permission set
+  description  = "An example Permission Set"                 // add desired description for permission set
+  // add multiple managed policies
+  managed_policies = [
+    "arn:aws:iam::aws:policy/AdministratorAccess",
```

Comment:
this admin policy contradicts with the view-only policy, for the example, I suggest to remove the Admin access

Reply:
updated and pushed changes 👍🏾